src/main/java/seedu/taskitty/model/ModelManager.java

```diff
@@ -131,6 +130,30 @@ public synchronized void removeUnchangedState() {
         historyPredicates.pop();
     }
     
+    private ReadOnlyTaskManager getPreviousTaskManager() {
+        return historyTaskManagers.pop();
+    }
+    
+    private Predicate getPreviousPredicate() {
+        return historyPredicates.pop();
+    }
+    
+    private String getPreviousCommand() {
+        return historyCommands.pop();
+    }
+    
+    private boolean noMorePreviousCommand () {
+        return historyCommands.isEmpty();
+    }
+    
+    //@@author A0135793W
+    private void updateFilters() {
```

Comment:
I think this is my method. It should be fine because this was removed when I was fixing the bug